trafilatura/xml.py

```diff
@@ -211,7 +211,8 @@ def validate_tei(xmldoc):  # , filename=""
 
 
 def replace_element_text(element, include_formatting):
-    '''Determine element text based on text and tail'''
+    '''Determine element text based on **just the text** of the element. You must deal with the tail separately.'''
+    elem_text = element.text
```

Comment:
Is there a good reason to define an additional variable instead of `element.text`?